stable/lamp/values.yaml

```diff
@@ -0,0 +1,337 @@
+  ## not create working clone or sync files
+  allowOverwrite: true
+
+  ## svn.resources resource requests/limits
+  resources: false
+  #  requests:
+  #    cpu: 1m
+  #    memory: 1Mi
+
+phpmyadmin:
+  ## phpmyadmin.enabled Enables phpmyadmin service
+  enabled: false
+
+  ## phpmyadmin.port Port to advertise service in LoadBalancer mode
+  port: 8080
+
+  ## phpmyadmin.subdomain Subdomain to advertise service on if ingress is enabled
+  subdomain: phpmyadmin
+
+  ## phpmyadmin.resources resource requests/limits
+  resources: false
+  #  requests:
+  #    cpu: 1m
+  #    memory: 1Mi
+
+resources:
+  requests:
+    ## resources.requests.cpu CPU resource requests
+    cpu: 1m
+
+    ## resources.requests.memory Memory resource requests
+    memory: 1Mi
+
+  # limits:
+  ## resources.limits.cpu CPU resource limits
+  #   cpu: 1000m
+
+  ## resources.limits.memory Memory resource limits
+  #   memory: 1024Mi
+
+
+### STORAGE ###
+
+persistence:
+  ## persistence.enabled Enables persistent volume - PV provisioner support necessary
+  enabled: true
+
+  ## persistence.keep Keep persistent volume after helm delete
+  keep: false
+
+  ## persistence.accessMode PVC Access Mode
+  accessMode: ReadWriteOnce
+
+  ## persistence.size PVC Size
+  size: 5Gi
+
+  ## persistence.storageClass PVC Storage Class
+  ## If defined, storageClassName: <storageClass>
+  ## If set to "-", storageClassName: "", which disables dynamic provisioning
+  ## If undefined (the default) or set to null, no storageClassName spec is
+  ##   set, choosing the default provisioner.  (gp2 on AWS, standard on
+  ##   GKE, AWS & OpenStack)
+  ##
+  # storageClass: "-"
+
+  ## persistence.hostPath if specified, used as persistent storage instead of PVC
+  # hostPath: /path/to/volume
+
+
+### NETWORKING ###
+
+service:
+  ## service.type Changes to ClusterIP automatically if ingress enabled
+  type: LoadBalancer
+
+  ## service.HTTPPort Port to advertise the main web service in LoadBalancer mode
+  HTTPPort: 80
+
+
+ingress:
+  ## ingress.enabled Enables ingress support - working ingress controller necessary
+  enabled: false
+
+  ## ingress.domain domain to advertise the services - A records need to point to
+  ## ingress controllers IP
+  # domain: example.com
+
+  ## ingress.subdomainWWW enables www subdomain and 301 redirect from domain
+  # subdomainWWW: false
+
+  ## ingress.ssl Enables [lego](https://github.com/jetstack/kube-lego) letsencrypt
+  ## ssl support - working lego container necessary
+  # ssl: false
+
+  ## ingress.htpasswdString if specified main web service requires authentication.
+  ## Format: _user:$apr1$F..._
+  # htpasswdString: "user:$apr1$FfVI2PRr$f9MW8rsr5RmmxbdV0Iipk1"
+
+  ## ingress.annotations specify custom ingress annotations such as e.g.
+  ## `ingress.kubernetes.io/proxy-body-size
+  annotations:
+    ingress.kubernetes.io/proxy-body-size: "50m"
+    kubernetes.io/ingress.class: nginx
```

Comment:
Should leave these commented so that you can use the default ingress in the cluster, unless the proxy-body-size and NGINX is required.

Reply:
Indeed, nginx ingress controller is not mandatory although some features such as ssl are only working with nginx controller. I've commented that part out and added explanations regarding nginx controller requirements to the values which need the controller.